lab2/css/style.scss

```diff
@@ -0,0 +1,506 @@
+$mainnewsbg: #f7f9fa;

```

Comment:
Hello there)) First of all, could you please split this file into several files based on such entities as upper, mainnav, mainnews-wrapper, video-side, omgnews-wrapper, misc and footer? Also, I would like if each media query will be placed next to the selector. For example
```
.elem {
  width: 200px;

  @media (min-width: $desktop-bp) {
    &:not(.active) {
      width: auto; // or whatever
    }
  }
}
```